pkg/lifecycle/lifecycle.go

```diff
@@ -2,15 +2,17 @@ package lifecycle
 
 import (
 	"errors"
-	"fmt"
 	"os"
-	"os/exec"
 
 	"github.com/go-git/go-billy/v5"
 	"github.com/rancher/charts-build-scripts/pkg/filesystem"
 	"github.com/rancher/charts-build-scripts/pkg/git"
 	"github.com/rancher/charts-build-scripts/pkg/path"
-	"github.com/sirupsen/logrus"
+)
+
+var (
+	errorChartRepository error = errors.New("chart repository is in an inconsistent state")
```

Comment:
You don't have to indicate type here. The compiler will infer it. The name should be `errChartRepository`.